ultet/src/main/scala/za/co/absa/ultet/dbitems/DBFunctionFromPG.scala

```diff
@@ -16,19 +16,61 @@
 package za.co.absa.ultet.dbitems
 
 
-import za.co.absa.ultet.model.function.{FunctionArgumentType, FunctionName}
-import za.co.absa.ultet.model.{DatabaseName, SQLEntry, SchemaName, UserName}
+import za.co.absa.ultet.model.function.{FunctionArgumentType, FunctionDrop, FunctionName}
+import za.co.absa.ultet.model.{DatabaseName, SQLEntry, SchemaName}
 
-case class DBFunctionFromPG(sqlEntries: Seq[SQLEntry]) extends DBFunction {
-  override def fnName: FunctionName = ???
+import java.sql.Connection
 
-  override def paramTypes: Seq[FunctionArgumentType] = ???
+case class DBFunctionFromPG(
+                             schema: SchemaName,
+                             fnName: FunctionName,
+                             paramTypes: Seq[FunctionArgumentType]
+                           ) extends DBFunction {
+  override def sqlEntries: Seq[SQLEntry] = Seq(FunctionDrop(schema, fnName, paramTypes))
+}
+
+object DBFunctionFromPG {
+
+  def fetchAll(schemaName: SchemaName)
```

Comment:
Would name it `fetchAllOfSchema` or something like that.

Reply:
done